app/Jobs/SendReviewedPostToCIO.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Rogue\Jobs;
+
+use Rogue\Models\Post;
+use Illuminate\Bus\Queueable;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+
+class SendReviewedPostToCIO implements ShouldQueue
```

Comment:
This is kinda nitpicky, but I think this might read better as `SendReviewedPostToCustomerIo` (so you don't need to figure out what `CIO` stands for). If we do keep the abbreviation, Laravel's style rules also write them title-case (like `Http`). :neckbeard:

Reply:
I'm down with spelling it out!

Reply:
@DFurnes updated! cool to merge?

Reply:
Good by me! ⛵️